access-mgmt/README.adoc

```diff
@@ -0,0 +1,473 @@
+
+//Additional line break to resolve mod docs generation error, not sure why. Leaving for now. (Stetson, 20 May 2021)
+
+[id="con-access-management_{context}"]
+== Access management in {product}
+
+[role="_abstract"]
+{product} uses Access Control Lists (ACLs) provided by Kafka that enable an organization administrator to manage how user accounts and service accounts are permitted to interact with Kafka resources, such as instances and topics. A user account enables users in your organization to access resources. A service account enables your application or tool to connect securely to your resources.
+
+Administrators can allow or deny operations that specific accounts or that all accounts in the organization can perform for Kafka resources. A resource in an ACL can be a Kafka instance, topic, consumer group, or producer transaction. Administrators can also allow or deny access for other users to define account permissions for an instance.
+
+Users who aren't administrators can fully access a Kafka instance that they create and can manage the level of access for service accounts that they create for their own instances. However, only administrators can manage the level of access of other users or of service accounts created by other users for a Kafka instance.
+
+Administrators can define ACLs for the following components:
+
+* A single account or all accounts within the organization
+* A single named resource, all resources of a particular type (such as a topic, consumer group, or transactional ID), or all resources of a particular type with a specified prefix
+* A single operation (such as `Write`) for a resource or all operations for a resource
+
+If two or more permission settings in an ACL match a request being made to the Kafka broker and at least one of the matching permissions specifies that the action is denied, then the request is denied.
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://kafka.apache.org/documentation/#security_authz[Authorization and ACLs^] in Kafka documentation
+
+=== Supported ACL permissions in {product}
+
+An ACL acts as a mapping of permitted operations on specified resources for a selected account or for all accounts in an organization. An account can be a user account or a service account. Operations correspond to one or more Kafka APIs or request types that are applicable for the specified resource.
+
+For example, a `Read` operation for a `Topic` resource corresponds to the `Fetch`, `OffsetCommit`, and `TxnOffsetCommit` Kafka requests. A `Write` operation for a `Topic` resource corresponds to the `Produce` and `AddPartitionsToTxn` Kafka requests.
```

Comment:
This is almost a verbatim copy of the text from Confluent, without a reference/citation to the original text: https://docs.confluent.io/platform/current/kafka/authorization.html

I'm not sure what the license is on those docs, and therefore I don't know if we can simply copy their text, but just wanted to point this out to avoid potential problems.

Reply:
@DuncanDoyle Good point. I wasn't actually trying to randomly copy Confluent, they just have exactly what I was going for, so I did copy, but just as a basis. I wanted to point out that each operation corresponds to Kafka APIs or request types that relate to the specified resource, so that users have a bit more context about the operations and how they relate back to Kafka. I also wanted to give a couple of examples and read and write are the most common so I wanted those.

Anyway, I've re-modified the previous paragraph a bit more to distinguish. As per the examples, I believe it's different enough to be okay, imo. I don't know what more to change than I already have, and I want to make the examples clear. And I don't want to refer to Confluent because I'm already referring to Kafka, which is the true source.